stdlib/LinearAlgebra/src/adjtrans.jl

```diff
@@ -452,3 +452,15 @@ pinv(v::TransposeAbsVec, tol::Real = 0) = pinv(conj(v.parent)).parent
 ## complex conjugate
 conj(A::Transpose) = adjoint(A.parent)
 conj(A::Adjoint) = transpose(A.parent)
+
+## reductions
+for (ttype, transform) in ((:Adjoint, adjoint), (:Transpose, transpose))
+    @eval _mapreduce(f, op, ::IndexStyle, A::$ttype) =
+        _mapreduce(f∘$transform, op, IndexStyle(parent(A)), parent(A))
```

Comment:
i think this should not overwrite the IndexStyle. instead the default index style function should return the default index style of the parent

Reply:
I think this method definition should be removed anyway, as the above cited ones should do the job already.